guides/common/modules/proc_configuring-logging-to-journal-or-file-based-logging.adoc

```diff
@@ -0,0 +1,43 @@
+[id="Configuring_Logging_to_Journal_or_File-based_Logging_{context}"]
+= Configuring Logging to Journal or File-based Logging
+
+{Project} uses file-based logging by default.
+You can use the `{foreman-installer}` command to reconfigure logging.
+
+.Procedure for configuring logging with Journal
+. Enter the following `{foreman-installer}` command to configure logging to the `journald` service:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# {foreman-installer} \
+--foreman-logging-layout pattern \
+--foreman-logging-type journald \
+--foreman-proxy-log JOURNAL
+----
+
+. Optional: To inspect the log messages, use the `journalctl` utility.
+For example:
++
+* `journalctl -u foreman` and `journalctl -u foreman-proxy` show messages for the `foreman` and `foreman-proxy` units
```

Comment:
Should we use `--unit` instead of `-u` in the example for a self-documenting parameter?

Reply:
Thanks, applied.